Code/GraphMol/Conformer.h

```diff
@@ -71,7 +70,10 @@ class RDKIT_GRAPHMOL_EXPORT Conformer : public RDProps {
   //! Reserve more space for atom position
   void reserve(unsigned int size) { d_positions.reserve(size); }
 
-  //! Get the molecule that oqns this conformation
+  //! returns whether or not this instance belongs to a molecule
+  bool hasOwningMol() const { return dp_mol != nullptr; };
+
+  //! Get the molecule that owns this instance
   ROMol &getOwningMol() const { return *dp_mol; }
```

Comment:
Needs PRECONDITION now that we have hasOwningMol, i.e. this implies that we may not have an owning mol :)

Reply:
yikes. You are completely correct.

Reply:
Ok, add the `PRECONDITION` here and in `SubstanceGroup`